app/helpers/job_invocation_output_helper.rb

```diff
@@ -0,0 +1,33 @@
+module JobInvocationOutputHelper
+
+  def colorize_line(line)
```

Comment:
oh look i already created an issue for it :) http://projects.theforeman.org/issues/12105